tests/phpunit/elementor/core/upgrade/test-upgrades.php

```diff
@@ -453,4 +478,50 @@ private function create_image() {
 	private function delete_image( $attachment_id ) {
 		wp_delete_attachment( $attachment_id, true );
 	}
+
+	/**
+	 * @param $documents
+	 *
+	 * @return void
+	 */
+	private function assert_flex_gap_control_has_changed( $elementor_data ) {
+		$top_level_container = $elementor_data[0];
+		self::assertEquals( [
```

Comment:
is there any specific purpose to access assertion function through `self::` in non-static method? 
Let's do it using `$this` like in `test_v_3_15_9_container_updates`

Reply:
Correct
Fixed